<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>155</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>251</width>
    <height>155</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>251</width>
    <height>155</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="euroValueLabel">
            <property name="text">
             <string>Курс евро</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="euroValueEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="priceLabel">
            <property name="text">
             <string>Общая цена</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="priceEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="questionMark">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resources.qrc">:/new/icons/question-mark.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="button">
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: black; border: none;
	font: 87 15pt &quot;Segoe UI Black&quot;;
	color: white;
	border-radius: 5px;
}
QPushButton:pressed{
	border: 3px solid;
	border-color: #a20379;
	padding: 0px;
}</string>
        </property>
        <property name="text">
         <string>Рассчитать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="priceLabel2">
          <property name="text">
           <string>Цена в рублях:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="priceValueLabel">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
